Backend/src/server.ts

```diff
@@ -26,6 +26,8 @@ app.use(passport.initialize());
 app.use(requestLogger);
 //The regular logger needs to be before the router
 
+const tokenExpirationInSeconds = 60;
```

Comment:
Should probably be changed before merge. This is in seconds. For 1 hour you can replace it with a string like "1h".

> Eg: 60, "2 days", "10h", "7d". A numeric value is interpreted as a seconds count. If you use a string be sure you provide the time units (days, hours, etc), otherwise milliseconds unit is used by default ("120" is equal to "120ms").

If int, it is seconds. 

```suggestion
const tokenExpirationInSeconds = "1h";
```